packages/contracts/contracts/DependencyRegistryV0.sol

```diff
@@ -104,12 +107,28 @@ contract DependencyRegistryV0 is
      * @notice Initializes contract.
      * @param adminACLContract_ Address of admin access control contract, to be
      * set as contract owner.
+     * @param coreRegistryAddress_ Address of core registry contract.
      */
-    function initialize(address adminACLContract_) public initializer {
+    function initialize(
+        address adminACLContract_,
+        address coreRegistryAddress_
+    ) public initializer {
```

Comment:
the initialize function cannot be called after an upgrade.
we have two options:
* define a new function with modifier `reinitializer(2)` and make it public, and call it atomically when upgrading via multisig call to `ProxyAdmin.upgradeAndCall` (we would need to build our bytes manually)
* skip reinitializer abstraction, and rely on the `updateCoreRegistryAddress`, while accepting some minor downtime on the dep registry until we update core registry address via `function updateCoreRegistryAddress`

I vote for the latter, which means initialize can remain unchanged in this PR

Reply:
thanks for flagging 🙏  went with the latter and reverted the changes to the initialize function.. is there any documentation i should update so we don't forget to call `updateCoreRegistryAddress`?

Reply:
I wouldn't worry about it too much, since we only have to do it once on the upcoming upgrade 👍

Reply:
sounds great :D